src/Microsoft.Health.Dicom.Core.UnitTests/Features/CustomTag/GetCustomTagsServiceTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Dicom;
+using Microsoft.Health.Dicom.Core.Features.Common;
+using Microsoft.Health.Dicom.Core.Features.CustomTag;
+using Microsoft.Health.Dicom.Core.Messages.CustomTag;
+using NSubstitute;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Core.UnitTests.Features.CustomTag
+{
+    public class GetCustomTagsServiceTests
+    {
+        private ICustomTagStore _customTagStore;
+        private IDicomTagParser _dicomTagParser;
+        private IGetCustomTagsService _getCustomTagsService;
+
+        public GetCustomTagsServiceTests()
+        {
+            _customTagStore = Substitute.For<ICustomTagStore>();
+            _dicomTagParser = Substitute.For<IDicomTagParser>();
+            _getCustomTagsService = new GetCustomTagsService(_customTagStore, _dicomTagParser);
+        }
+
+        [Fact]
+        public async Task GivenRequestForAllTags_WhenNoTagsAreStored_ThenEmptyListShouldBeReturned()
+        {
+            _customTagStore.GetAllCustomTagsAsync(default).Returns(new List<CustomTagEntry>());
+            GetAllCustomTagsResponse response = await _getCustomTagsService.GetAllCustomTagsAsync(default);
+
+            Assert.True(response.CustomTags.Count() == 0);
```

Comment:
>True [](start = 19, length = 4)

nit: use `Equals` instead so the error message if it fails is easier to understand. Otherwise all we'll see is "Assert failed" lol